jobs/integr8ly/installation-smoke-tests.yaml

```diff
@@ -37,4 +28,27 @@
           name: RELEASE_VERSION
           default: ''
           description: 'Provide release version when testing particular release.'
-    
\ No newline at end of file
+    dsl: |
+        node('cirhos_rhel7'){
+            stage('Clone QE repository') {
+                dir('.') {
+                    git branch: "${BRANCH}", url: "${REPOSITORY}"
+                }
+            }
+            stage('Run the smoke tests'){
+                dir('tests'){
+                    sh '''
+                        npm install
+                    '''
+                }
+                dir('tests/backend-testsuite'){    
+                    try {
+                        sh '''
+                            gulp smoke
+                        '''
+                    } catch(exp) {
+                        currentBuild.result = 'UNSTABLE'
```

Comment:
btw, @tremes @trepel do you guys know what would be the result of the build if we didn't catch failure here and just let sh/gulp exit with non-zero code?

Reply:
The result is FAILURE in such case

Reply:
Why do we prefer UNSTABLE instead of FAILURE?

Reply:
No strong opinion here. Would you prefer FAILURE?

Reply:
I don't have a strong opinion about the state. But wrapping it in try/catch seemed useless to me at first.

P.S: This is kinda of topic for this PR, sorry for starting this conversation.

Reply:
Yes this was kinda my experiment TBH...and you're probably right that's useless (just yellow color instead of red...from time to time:-))

Reply:
@matskiv @tremes any suggestions about the try/catch, so the PR can be merged?

Reply:
removing try/catch block in such case and merging